Userland/Applications/PixelPaint/MainWidget.cpp

```diff
@@ -876,6 +876,16 @@ void MainWidget::initialize_menubar(GUI::Window& window)
         }
     }));
 
+    auto& scopes_menu = window.add_menu("&Scopes");
```

Comment:
I'd expect toggles to show/hide certain parts of the UI to be under the menu "View" instead.

Reply:
Makes sense. I will move this to under the view menu. Do you think it still makes sense for the Scopes menu to be there though, like View -> Scopes -> toggles, or just flatten it to View -> Vectorscope and View -> Histogram?

Reply:
The former - if we add more scopes, it makes sense to have that submenu already in place.